tools/test-case-generators/repos.json

```diff
@@ -325,6 +325,14 @@
         "image_type_tags": [
           "ec2"
         ]
+      },
+      {
+        "name": "rhui-azure",
+        "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/rhvpn/el9/el9-x86_64-rhui-azure-20221015",
```

Comment:
Is it possible to put comments in this json file (and have them preserved)? Having these mis-named is going to confuse the heck out of someone (me!) eventually...

Reply:
Not really.  I guess we could add a key called "comment" and add the comment as a value and our tools would ignore it.
What we should actually do though is define a new rpmrepo snapshot for aarch64 with the same source.  It wont take up any more space in the bucket, it would just be extra metadata for the new endpoint and it would make things clearer and predictable.